packages/runtime/test-runtime-utils/src/mocksDataStoreContext.ts

```diff
@@ -37,30 +37,30 @@ import { v4 as uuid } from "uuid";
 import { IDocumentStorageService } from "@fluidframework/driver-definitions";
 
 export class MockFluidDataStoreContext implements IFluidDataStoreContext {
-    public documentId: string;
+    public documentId: string = undefined as any;
```

Comment:
Please log a bug to remove this.  This was removed from the IFluidDataStoreContext interface a while ago.  I don't this is not necessary.

But it is a "breaking" change, so probably don't want to do it with this change.

Reply:
Should such a bug be filed in Azure DevOps? I will speak with Abram tomorrow about how to do so.